lib/cocoapods/command/outdated.rb

```diff
@@ -90,7 +93,7 @@ def deprecated_pods
 
       def spec_sets
         @spec_sets ||= begin
-          analyzer.send(:update_repositories) unless config.skip_repo_update?
+          analyzer.send(:update_repositories) if repo_update?(:default => true)
```

Comment:
I think it makes sense to have `pod outdated` do the repo update - I really doubt it has the level of usage that `install` / `update` has